stylesheets/modules/mozugrid/grid.less

```diff
@@ -1,55 +1,71 @@
-.mz-grid{
+.mz-grid {
     width: 100%;
     margin: 15px 0;
     border: 1px solid #999;
+
```

Comment:
So many whitespace changes. Please revert this is a public repo. Thanks.

Reply:
It is very difficult to revert  in vs2019. so reverted in notepad.
When  I added Bhushan's changes, It autoformats document on paste.  need to find solution to this problem

Reply:
Thanks @rajeshdatir. I did this again since Bhushan's commit caused the same issue. @SharmaBhushan please be careful with white space/formatting only changes. I recommend you disable format on paste/save since it can reformat the entire file.

Reply:
Sure @michaelsadlonkibo I will take care. Thanks